### PR TITLE
Set up project

### PR DESCRIPTION
**Pull request by lucabenaventew**
# Project Updates :green_circle: 
- [ ] Install Ruby on Rails framework.
- [ ] Use Ruby Gems as a software packages system.
- [ ] Understand what ORM is.
- [ ] Use database migration files to maintain database schema.
- [ ] Install and check linters.

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Luca, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, with a ReadMe that does not conform to the **new** [standardized ReadMe template](https://github.com/microverseinc/readme-template). Kindly ensure to update your ReadMe to this [standardized template](https://github.com/microverseinc/readme-template), and then request another review.

**[New ReadMe Template](https://github.com/microverseinc/readme-template):**
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: db/migrate/20221018225259_add_user_to_likes.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUserToLikes < ActiveRecord::Migration[7.0]
+    def change
+      add_reference :likes, :user, null: false, foreign_key: true
+    end
+  end
```
Comment (Whoistolu) [inline db/migrate/20221018225259_add_user_to_likes.rb:5]: - [ ] [OPTIONAL] Kindly remove this migration file as the project requirements only stated that you create tables for User, Post, Comment, and like only.

File: db/migrate/20221018225327_add_user_to_comments.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddUserToComments < ActiveRecord::Migration[7.0]
+    def change
+      add_reference :comments, :user, null: false, foreign_key: true
+    end
+  end
```
Comment (Whoistolu) [inline db/migrate/20221018225327_add_user_to_comments.rb:5]: - [ ] [OPTIONAL] Kindly remove this migration file as the project requirements only stated that you create tables for User, Post, Comment, and like only.

File: db/migrate/20221018225404_add_user_to_posts.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddUserToPosts < ActiveRecord::Migration[7.0]
+    def change
+      add_reference :posts, :user, null: false, foreign_key: true
+    end
+  end
```
Comment (Whoistolu) [inline db/migrate/20221018225404_add_user_to_posts.rb:5]: - [ ] [OPTIONAL] Kindly remove this migration file as the project requirements only stated that you create tables for User, Post, Comment, and like only.

File: db/migrate/20221018225443_add_post_to_likes.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class AddPostToLikes < ActiveRecord::Migration[7.0]
+    def change
+      add_reference :likes, :post, null: false, foreign_key: true
+    end
+  end
```
Comment (Whoistolu) [inline db/migrate/20221018225443_add_post_to_likes.rb:5]: - [ ] [OPTIONAL] Kindly remove this migration file as the project requirements only stated that you create tables for User, Post, Comment, and like only.

File: db/migrate/20221018225521_add_post_to_comments.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+class AddPostToComments < ActiveRecord::Migration[7.0]
+    def change
+      add_reference :comments, :post, null: false, foreign_key: true
+    end
+  end
```
Comment (Whoistolu) [inline db/migrate/20221018225521_add_post_to_comments.rb:5]: - [ ] [OPTIONAL] Kindly remove this migration file as the project requirements only stated that you create tables for User, Post, Comment, and like only.

**Review by Whoistolu — APPROVED** [review]
### Hi @lucabenaventew 

Great job using the required README template as recommended by the previous reviewer 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — COMMENTED** [review]
#  *Status: Invalid Request* :red_circle: 

Hi @lucabenaventew, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] has no professional README. Kindly provide appropriate information under each of the headings in your README file.
- [ ] The link to the MIT License on your README file is broken. Kindly follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to set it up properly

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**